monitoring/mongodb/alerts.yaml

```diff
@@ -183,10 +183,10 @@ groups:
 
   - alert: MongoDbRSNotSynced
     expr: |
-      sum by (rs_nm) (mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*", member_state="SECONDARY"}) != (${replicas} - 1)
+      floor(avg by(rs_nm)(count by (rs_nm, pod)(mongodb_rs_members_state{namespace="${namespace}", pod=~"${service}.*", member_state="SECONDARY"}))) != (${replicas} - 1) 
```

Comment:
went for the floor avg as we want to know when the min or max is different from the expected secondaries number

Reply:
to ensure we don't _miss_ any alert, we should do the boolean condition before the final aggregation.
Otherwise, we may be unlucky and different pod metrics may average with the right value....

```
<aggregation operator> by(rs_nm) (
    count by(rs_nm, pod) (...) != ${replicas} - 1
)
```

→ if no `pod` triggers the alert, the aggregation will have nothing to aggregate, and no series will trigger the alert
→ if one (or more) `pod` triggers the alert, then the aggregation operation is used to filter-out the `pod` label, and (possibly, but not mandatory) generate an aggregate `value` for the alert (`min` / `avg` / `max` ... number of replicas : though there could always be corner cases where the resulting value is unexecpted... ; if we don't want a value, just using `group` should work)